src/__tests__/components/Dashboard/index.test.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
 import { mount } from 'enzyme';
 import Dashboard from '../../../components/Dashboard';
 
 describe('test dashboard', () => {
+  const state = {};
+  const props = {
+    currentUser: {
+      additionalUserInfo: {
+        profile: {
+          name: 'kevin koech',
+          picture: '',
+        },
+      },
+    },
+    removeCurrentUser: jest.fn(),
+    history: {
+      push: jest.fn(),
+    },
+  };
+  const mockStore = configureStore();
+  const store = mockStore(state);
   it('renders a card', () => {
-    // eslint-disable-next-line react/jsx-filename-extension
-    const wrapper = mount(<Dashboard />);
-    expect(wrapper).toMatchSnapshot();
-    wrapper.unmount();
+    const wrapper = mount(
```

Comment:
Nice work but I do not think this test really tests for rendering a card. There is no you have asserted that a card is displayed. I also think it is not necessary to pass state or props to this component more so when you omit them the test still passes.

Reply:
It actually does not test the rendering. This is a container for housing the other cards. The test will keep changing as the cards are added